cypress/e2e/inventory/search/browse-call-number-with-space.cy.js

```diff
@@ -1,54 +1,73 @@
-import getRandomPostfix from '../../../support/utils/stringTools';
-import DevTeams from '../../../support/dictionary/devTeams';
-import TestTypes from '../../../support/dictionary/testTypes';
-import InventoryActions from '../../../support/fragments/inventory/inventoryActions';
-import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
-import InventorySearchAndFilter from '../../../support/fragments/inventory/inventorySearchAndFilter';
-import BrowseCallNumber from '../../../support/fragments/inventory/search/browseCallNumber';
-import TopMenu from '../../../support/fragments/topMenu';
-import BrowseContributors from '../../../support/fragments/inventory/search/browseContributors';
+import getRandomPostfix from "../../../support/utils/stringTools";
```

Comment:
I think it's better to use single quotes)

Reply:
Yeah you are right) I forgot to change the settings of "Prettier" in vs code, it changed quotes from single to double automatically and other parts as well), I'll fix it thanks.